xbmc/music/MusicDatabase.cpp

```diff
@@ -7600,7 +7594,7 @@ static const translateJSONField JSONtoDBSong[] = {
   { "displayconductor",          "string", false, "Role_Conductor",         "song_artist.idRole AS Role_Conductor" },
   { "displayorchestra",          "string", false, "Role_Orchestra",         "song_artist.idRole AS Role_Orchestra" },
   { "displaylyricist",           "string", false, "Role_Lyricist",          "song_artist.idRole AS Role_Lyricist" },
-
+ 
```

Comment:
extra space

Reply:
I swear they secretly breed in my dev env while I'm not looking